WordPressVIPMinimum/Sniffs/VIP/SessionFunctionsUsageSniff.php

```diff
@@ -1,75 +0,0 @@
-<?php
-/**
- * WordPressVIPMinimum Coding Standard.
- *
- * @package VIPCS\WordPressVIPMinimum
- * @link https://github.com/Automattic/VIP-Coding-Standards
- * @license https://opensource.org/licenses/MIT MIT
- */
-
-namespace WordPressVIPMinimum\Sniffs\VIP;
-
-use WordPress\AbstractFunctionRestrictionsSniff;
-
-/**
- * Discourages the use of session functions.
- *
- * @link https://lobby.vip.wordpress.com/wordpress-com-documentation/code-review-what-we-look-for/#session-start-and-other-session-functions
- *
- * @package VIPCS\WordPressVIPMinimum
- *
- * @since   0.5.0
- */
-class SessionFunctionsUsageSniff extends AbstractFunctionRestrictionsSniff {
-
-	/**
-	 * Groups of functions to restrict.
-	 *
-	 * Example: groups => array(
-	 *  'lambda' => array(
-	 *      'type'      => 'error' | 'warning',
-	 *      'message'   => 'Use anonymous functions instead please!',
-	 *      'functions' => array( 'file_get_contents', 'create_function' ),
-	 *  )
-	 * )
-	 *
-	 * @return array
-	 */
-	public function getGroups() {
-		return [
-			'session' => [
```

Comment:
Where has this array been moved to? I don't see any Restricted*Sniff updated in this PR.

<img width="362" alt="screenshot 2019-01-04 at 23 55 57" src="https://user-images.githubusercontent.com/88371/50716996-5afc8b00-107c-11e9-8613-b21b8e796886.png">

This is why the [tests are failing](https://travis-ci.org/Automattic/VIP-Coding-Standards/jobs/475526016) :-)

Reply:
Sorry about that, this PR wasn't complete, I was mucking around with the rebasing at the time...